1_1.js

```diff
@@ -0,0 +1,4 @@
+function findKim(seoul){
+    var idx = seoul.indexOf('Kim');
+    return "김서방은 " + idx + "에 있다";
```

Comment:
```suggestion
    return `김서방은 ${idx}에 있다`;
```
백틱(``)과 ${}를 이용하는 템플릿 리터럴로 사용하는 법도 공부해 보세요

Reply:
감사합니다 훨씬깔끔해보여요